src/pep8ext_naming.py

```diff
@@ -275,6 +276,40 @@ def superclass_names(cls, name, parents: Iterable, _names=None):
                 names.update(cls.superclass_names(base.id, parents, names))
         return names
 
+    def visit_module(self, node, parents: Iterable, ignore=None):
+        for body in node.body:
+            try:
+                if len(body.targets) != 1:
+                    continue
+                name = body.targets[0].id
+                func_name = body.value.func.id
+                args = [a.value for a in body.value.args]
+                keywords = {kw.arg: kw.value.value for kw in body.value.keywords}
+            except AttributeError:
+                continue
+
+            if func_name != "TypeVar" or _ignored(name, ignore):
+                return
```

Comment:
I think this should be a `continue`?

```suggestion
                continue
```